doc/README.agda

```diff
@@ -113,7 +113,7 @@ import README.IO
 -- • Tactic
 --     Tactics for automatic proof generation
 
--- ∙ Text
+-- • Text
```

Comment:
what is the change here? A different unicode dot?

Reply:
Yes; this one was an oddball, the diff makes it consistent with other heading-level dots:
> -- • Level
-- • Reflection
-- • Relation
-- • Size
-- • Strict
-- ∙ Text        --- was
-- • Tactic   --- fixed